public/index.html

```diff
@@ -36,5 +41,8 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/react-bootstrap/3.3.7/react-bootstrap.min.js"></script>
```

Comment:
if you're includingreact-bootstrap here, there's no need to add it as a dependency, also still need to know if we can't use pure bootstrap

Reply:
I don't understand this one

Reply:
is it necessary to include this line? when you already installed it via npm

Reply:
Yes. The documentation requires it is included
https://react-bootstrap.github.io/getting-started/introduction/